src/stream/src/executor/exchange/output.rs

```diff
@@ -98,57 +34,38 @@ pub struct RemoteOutput {
     ch: Sender,
 }
 
-impl RemoteOutput {
+impl Output {
     pub fn new(actor_id: ActorId, ch: Sender) -> Self {
         Self {
             actor_id,
-            span: await_tree::span!("RemoteOutput (actor {:?})", actor_id).verbose(),
+            span: await_tree::span!("Output (actor {:?})", actor_id).verbose(),
```

Comment:
Shall we still preserve the information of `Local` or `Remote`? This can be helpful when debugging stuck caused by gRPC.

Reply:
Actually the purpose of this series of PRs is to avoid broadcasting and maintaining the global actor info, which is used in dispatch and merge executor. Previously in the dispatch executor side, we get the actor info only to distinguish between remote and local output. Now the difference between them is just the await tree span message, and it's not worth it to maintain the global actor info just to provide more detailed spawn message.

Besides, we provide the actor id in the span message, and we can actually infer whether it's remote or local output by checking if the downstream actor id is in the same CN as the current actor id. So we won't lose any debug information after this PR. It just make this information not straightforward, which I think it's acceptable.